docs/README.md

```diff
@@ -4,14 +4,19 @@ DGL document and tutorial folder
 Requirements
 ------------
 You need to build DGL locally first (as described [here](https://docs.dgl.ai/install/index.html#install-from-source)), and ensure the following python packages are installed:
+
```

Comment:
Could consider build a script for this task similar to https://github.com/dmlc/dgl/tree/master/script